### PR TITLE
chore: add proper breadcrumb nav to results

### LINKED ISSUE #88: results: update breadcrumb navigation
currently is just the `id` but we should have more info about the test in there

image

### PR DESCRIPTION
**Pull request by eric-hc**
- Breadcrumb navigation for results so user can navigate back to repository contents (closes #88)
- Also cleans up results layout a bit
- Finds unique test & test result across projects

### CODE REVIEW

File: src/pages/api/tests/index.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default async function handler(req: NextApiRequest, res) {
     case 'POST':
       const { repoId, path, repo } = JSON.parse(req.body)
       // is the test exist
-      const isTest = await test.findFirst({
+      const isTest = await test.findFirst({ // this should be findUnique and use wombo combo
```
Comment (eric-hc) [inline src/pages/api/tests/index.ts:12]: @tiffkwin @swand0g assist please ⛹️

File: src/components/nav-breadcrumb.jsx [hunk 2]
```diff
@@ -30,6 +30,43 @@ export const NavBreadcrumb = props => {
...
+        </BreadcrumbItem>
+        <BreadcrumbItem color="inherit">
+          <BreadcrumbLink href={`/projects/${owner}`}>
+            {owner}
+          </BreadcrumbLink>
+        </BreadcrumbItem>
+        {path.map((item, idx) => {
+          return (
+            <BreadcrumbItem color="inherit" key={idx}>
+              <BreadcrumbLink href={`/projects/${owner}/${path.slice(null, idx + 1).join('/')}`}>
```
Comment (swand0g) [inline src/components/nav-breadcrumb.jsx:55]: VERY READABLE

File: src/lib/prisma/schema.prisma [hunk 3]
```diff
@@ -71,6 +71,8 @@ model test {
   repo    repo     @relation(fields: [repoId], references: [id])
   repoId  Int
   results result[]
+
+  @@unique(fields: [path, repoId], name: "test_repo")
```
Comment (swand0g) [inline src/lib/prisma/schema.prisma:75]: Wait what is this

File: src/lib/prisma/schema.prisma [hunk 4]
```diff
@@ -81,4 +83,6 @@ model result {
   data      String?
   user      user     @relation(fields: [ghLogin], references: [ghLogin])
   ghLogin   String
+
+  @@unique(fields: [id, testId], name: "result_test")
```
Comment (swand0g) [inline src/lib/prisma/schema.prisma:87]: And dis

File: src/components/nav-breadcrumb.jsx (hunk 2 above)
Comment (eric-hc) [reply src/components/nav-breadcrumb.jsx:55]: why you heff to yell

**Review by swand0g — APPROVED** [review]

**Pull request merged**